src/new/wallet.js

```diff
@@ -1634,25 +1732,35 @@ class HathorWallet extends EventEmitter {
   }
 
   /**
-   * @typedef BaseTransactionResponse
-   * @property {{hash:string, index:number, data:Buffer}[]} inputs
-   * @property {{value:number, script:Buffer, tokenData:number, decodedScript:*}[]} outputs
-   * @property {number} version
-   * @property {number} weight
-   * @property {number} nonce
-   * @property {number} timestamp
-   * @property {string[]} parents
-   * @property {string[]} tokens
-   * @property {string} hash
-   * @property {*} _dataToSignCache
-   */
-
-  /**
-   * @typedef CreateNewTokenResponse
-   * @extends BaseTransactionResponse
-   * @property {string} name
-   * @property {string} symbol
-   */
+   * Create a new token for this wallet
```

Comment:
Actually you create the send transaction object for a new token creation

Reply:
Changed